aspnetcore/performance/caching/memory.md

```diff
@@ -1,164 +1,111 @@
+By [Rick Anderson](https://twitter.com/RickAndMSFT), [John Luo](https://github.com/JunTaoLuo), and [Steve Smith](http://ardalis.com)
 
 [View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/performance/caching/memory/sample)
 
 <a name=caching-basics></a>
 
-## Caching Basics
-
-Caching can dramatically improve the performance and scalability of ASP.NET applications, by eliminating unnecessary requests to external data sources for data that changes infrequently.
-
-> [!NOTE]
-> Caching in all forms (in-memory or distributed, including session state) involves making a copy of data in order to optimize performance. The copied data should be considered ephemeral - it could disappear at any time. Apps should be written to not depend on cached data, but use it when available.
-
-ASP.NET supports several different kinds of caches, the simplest of which is represented by the `IMemoryCache` interface, which represents a cache stored in the memory of the local web server.
-
-You should always write (and test!) your application such that it can use cached data if it's available, but otherwise will work correctly using the underlying data source.
-
-An in-memory cache is stored in the memory of a single server hosting an ASP.NET app. If an app is hosted by multiple servers in a web farm or cloud hosting environment, the servers may have different values in their local in-memory caches. Apps that will be hosted in server farms or on cloud hosting should use a [distributed cache](distributed.md) to avoid cache consistency problems.
-
->[!TIP]
-> A common use case for caching is data-driven navigation menus, which rarely change but are frequently read for display within an application. Caching results that do not vary often but which are requested frequently can greatly improve performance by reducing round trips to out of process data stores and unnecessary computation.
-
-## Configuring In Memory Caching
-
-To use an in memory cache in your ASP.NET application, add the following dependencies to your *project.json* file:
-
-[!code-json[Main](memory/sample/src/CachingSample/project.json?range=7-13&highlight=4)]
-
-Caching in ASP.NET Core is a *service* that should be referenced from your application by [Dependency Injection](../../fundamentals/dependency-injection.md). To register the caching service and make it available within your app, add the following line to your `ConfigureServices` method in `Startup`:
+## Caching basics
 
-[!code-csharp[Main](memory/sample/src/CachingSample/Startup.cs?range=12-15&highlight=3)]
+Caching can significantly improve the performance and scalability of an app by reducing round trips to the data store. Caching works best with data that changes infrequently. Caching makes a copy of data that can be returned much faster than from the original source. You should write and test your app to never depend on cached data.
```

Comment:
Might want to elaborate on why you should never depend on cached data. The old docs mentioned that the data should be considered ephemeral. Maybe keep something along those lines?